src/DynamoCore/Utilities/Configurations.cs

```diff
@@ -208,8 +208,7 @@ public class Configurations
         public const string CategoryGroupAction = "Actions";
         public const string CategoryGroupQuery = "Query";
         public const string CategoryDelimiterString = ".";
-        public const char ShortenedCategoryDelimiter = '-';
-        public const string CategoryDelimiterWithSpaces = " - ";
```

Comment:
There is an extra space between ‘-‘ and ‘Create’ under search results.
![image](https://cloud.githubusercontent.com/assets/8158404/6687109/401f363e-ccb0-11e4-8c8c-2168284a2b2d.png)

CategoryDelimiterWithSpaces was used to create this type of space. I removed it.